swarm/runtime/_claude_sdk/hooks.py

```
@@ -0,0 +1,297 @@
+        success: bool,
+        context: Dict[str, Any],
+    ) -> None:
+        if tool_name not in ("Read", "Write", "Edit"):
+            return
+
+        file_path = tool_input.get("file_path", tool_input.get("path", ""))
+        entry = {
+            "tool": tool_name,
+            "path": file_path,
+            "success": success,
+            "timestamp": datetime.now(timezone.utc).isoformat() + "Z",
```

Ruff (0.14.11)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)